index.js

```diff
@@ -67,7 +68,144 @@ function getDefaultConfig(mainTemplate, templatesPath) {
 
       // Add extra function for finding a security scheme by name
       ramlObj.securitySchemeWithName = function (name) {
-        return ramlObj.securitySchemes.find((scheme) => scheme[name]);
+        for (var index = 0; index < ramlObj.securitySchemes.length; ++index) {
```

Comment:
Yeah so there was a bug somewhere, because it should :) I'll look into it today.